calico_containers/pycalico/netns.py

```diff
@@ -51,6 +52,55 @@ def setup_logging(logfile, level=logging.INFO):
     _log.addHandler(handler)
 
 
+def increment_metrics(namespace):
+    """
+    Increment the metric of any default route that has a metric of 0.
+    :param namespace: The Networking namespace of the container.
+    :return: None. Raises CalledProcessError on error.
+    """
+    def get_metric(route):
+        """
+        Gets the metric of a route.
+        :param route: String representation of a route (full line returned by
+        `ip route`.
+        :return: Integer value representing the metric of the route.
+        """
+        match = re.search('metric\s+(\d)', route)
```

Comment:
doesn't `\d` match only a single digit?  Metrics are unsigned 32 bit numbers, so you'll handle multi-digit numbers.  Here and on line 96.